Tests/ApolloSQLiteTests/CachePersistenceTests.swift

```diff
@@ -8,6 +8,36 @@ import SQLite
 
 class CachePersistenceTests: XCTestCase {
 
+  func testDatabaseSetup() throws {
+    // Create a database with a state corresponding to the first schema version.
+    let sqliteFileURL = SQLiteTestCacheProvider.temporarySQLiteFileURL()
+    let db = try Connection(.uri(sqliteFileURL.absoluteString), readonly: false)
+    try db.run("""
+    CREATE TABLE IF NOT EXISTS "records" ("_id" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "key" TEXT NOT NULL UNIQUE, "record" TEXT NOT NULL)
+    """)
+    try db.run("""
+    CREATE UNIQUE INDEX IF NOT EXISTS "index_records_on_key" ON "records" ("key")
+    """)
+
+    // Add en entry
+    try db.run("""
+    INSERT OR REPLACE INTO "records" ("key", "record") VALUES ("QUERY_ROOT.hero", "{""name"":""Luke Skywalker"",""__typename"":""Human""}")
+    """)
```

Comment:
One thing I've found in the past that can be somewhat less error prone than doing this setup in a test yourself is to provide an actual `.sqlite` file from an old version of the database that you can then make a copy of and test migrations on. That way you guarantee you're using the format that is on older versions.

Reply:
Done, let me know what you think

Reply:
I absolutely love it. Thank you!